libs/ui/elements/src/lib/metadata-info/metadata-info.component.ts

```diff
@@ -49,6 +49,18 @@ export class MetadataInfoComponent {
     }
   }
 
+  get getThemes() {
```

Comment:
I don't think these themes (like "cadastre, aménagement" in the example) actually need to be split (cc @jahow @fgravin ). I'm noticing that clicking on (unsplit) themes, but also "constructions et ouvrages" for example does not return any results in the search, however. Do we need to add them to the search request or index?

Reply:
I did split them because indeed, clicking on "unsplit" themes didn't return results while clicking on "split" ones did. But I can undo that if it's not the wanted behavior.